.grabl/automation.yml

```diff
@@ -135,8 +135,8 @@ build:
         sudo systemctl daemon-reload
         sudo systemctl start typedb
 
-        bazel run //test/example/python:phone-calls
-        bazel run //test/example/python:social-network
+        bazel run //test/example/python:phone-calls --action_env=PATH
+        bazel run //test/example/python:social-network --action_env=PATH
```

Comment:
The global python being set by pyenv was not being respected by bazel. Adding `--action_env` makes bazel respect the user-defined `PATH` and therefore find python of the version set by pyenv.